packages/wallet/src/Delegation/withdrawal.ts

```diff
@@ -0,0 +1,20 @@
+import { Lovelace } from '@cardano-ogmios/schema';
+import { Cardano, CardanoSerializationLib, CSL } from '@cardano-sdk/core';
+import { KeyManager } from '../KeyManagement';
+
+export type Withdrawal = {
+  address: CSL.RewardAddress;
+  quantity: CSL.BigNum;
+};
+
+export const withdrawal = (
+  csl: CardanoSerializationLib,
+  keyManager: KeyManager,
+  quantity: Lovelace,
+  network: number = Cardano.NetworkId.mainnet
+): Withdrawal => ({
+  // Review: are withdrawals supposed to go to stake address or
```

Comment:
- Funds cannot be sent by network participants to reward addresses, only the protocol can credit these accounts.
- We don't encourage wallets to implement an explicit _withdrawal_ action, as it serves no purpose and costs fees. Instead, reward accounts are simply considered funding sources for regular payment transactions in addition to UTxOs, and wallets should design a UX to withdraw all available rewards when making a payment transaction, returning the remaining as change.